primitives/src/sequence.rs

```diff
@@ -234,3 +234,38 @@ impl fmt::Debug for Sequence {
 
 #[cfg(feature = "alloc")]
 units::impl_parse_str_from_int_infallible!(Sequence, u32, from_consensus);
+
+#[cfg(test)]
+mod tests {
```

Comment:
Perhaps just cfc on the `alloc` feature here instead of each test?

Reply:
yep makes sense, updated